src/ui/TopicExplorer.svelte

```diff
@@ -2,9 +2,33 @@
   import ONSAccordion from "./../ui/ons/ONSAccordion.svelte";
   import ONSAccordionPanel from "./../ui/ons/partials/ONSAccordionPanel.svelte";
   import censusData from "./../data/simpleTopicTableCategoryData";
+  import { onMount } from "svelte";
   import slugify from "slugify";
   import { selectedData } from "../model/censusdata/censusdata";
 
+  export let selectedTopic;
+
+  let topicIndex;
+
+  $: {
+    if (selectedTopic) {
+      censusData.forEach((topic) => {
+        if (slugify(topic.name).toLowerCase() == selectedTopic.toLowerCase()) {
+          topicIndex = censusData.indexOf(topic);
+        }
+      });
+    }
+  }
+
+  // !!! Temporary solution -  to be removed when we'll be able to import the DS js bundle at a component level
```

Comment:
Just FYI, it's handy to stick to the [somewhat standard conventions](https://en.wikipedia.org/wiki/Comment_%28computer_programming%29#Tags) so we can grab all these comments easily later.